src/lib/support/Pool.h

```diff
@@ -323,6 +324,18 @@ class BitMapObjectPool : public internal::StaticAllocatorBitmap
         return ForEachActiveObjectInner(&proxy, &internal::LambdaProxy<T, Function>::ConstCall);
     }
 
+    void DumpToLog() const
+    {
+        ChipLogError(Support, "BitMapObjectPool: %lu allocated", static_cast<unsigned long>(Allocated()));
+        if constexpr (IsDumpable<T>::value)
+        {
+            ForEachActiveObjectInner(nullptr, [](void * context, const void * object) {
```

Comment:
Any reason to not just use ForEachActiveObject and avoid the casting needed?

Reply:
I was wondering if that whole LambdaProxy stuff would cause extra overhead, but probably not, and it won't matter before a crash anyway.